src/timelord/timelord.py

```diff
@@ -72,9 +72,12 @@ def __init__(self, root_path, config: Dict, constants: ConsensusConstants):
         # For each iteration submitted, know if it's a signage point, an infusion point or an end of slot.
         self.iteration_to_proof_type: Dict[uint64, IterationType] = {}
         # List of proofs finished.
-        self.proofs_finished: List[Tuple[Chain, VDFInfo, VDFProof]] = []
+        self.proofs_finished: List[Tuple[Chain, VDFInfo, VDFProof, int]] = []
```

Comment:
Do you mind making a separate PR with these timelord changes? 
BTW with pycharm you can easily switch between branches with the git GUI. It's very convenient.